src/int/neg.rs

```diff
@@ -0,0 +1,112 @@
+//! [`Int`] negation-related operations.
+
+use subtle::CtOption;
+
+use crate::{ConstChoice, Int, Word};
+
+impl<const LIMBS: usize> Int<LIMBS> {
+    /// Perform the two's complement "negate" operation on this [`Int`]:
+    /// map `self` to `(self ^ 1111...1111) + 0000...0001` and return the carry.
+    ///
+    /// Note: a zero carry indicates `self == Self::ZERO`.
+    ///
+    /// Warning: this operation is unsafe to use as negation; the negation is incorrect when
+    /// `self == Self::MIN`.
+    #[inline]
+    pub(crate) const fn negc(&self) -> (Self, Word) {
```

Comment:
Per the conventions currently used by `core` (see #693), perhaps this should be `carrying_neg`?

```suggestion
    /// Perform the two's complement "negate" operation on this [`Int`]:
    /// map `self` to `(self ^ 1111...1111) + 0000...0001` and return the carry.
    ///
    /// Note: a zero carry indicates `self == Self::ZERO`.
    ///
    /// Warning: this operation is unsafe to use as negation; the negation is incorrect when
    /// `self == Self::MIN`.
    #[inline]
    pub(crate) const fn carrying_neg(&self) -> (Self, Word) {
```

Reply:
Done.